x-pack/docs/en/rest-api/security/get-builtin-privileges.asciidoc

```diff
@@ -67,7 +67,7 @@ A successful call returns an object with "cluster" and "index" fields.
     "manage",
     "manage_api_key",
     "manage_ccr",
-    "manage_data_frame_transforms",
+    "manage_transform",
```

Comment:
this should not be changed, manage/monitor transform are already here.

CI should test this.